### PR TITLE
Update volume encryption doc

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn/longhorn#4883

### CODE REVIEW

**Comment by innobead** [comment]
@jillian-maroket please help review this. Thanks.

**Review by innobead — APPROVED** [review]

File: content/docs/1.6.0/advanced-resources/security/volume-encryption.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ title: Volume Encryption
 weight: 2
 ---
 
-Longhorn supports encrypted volumes by utilizing the linux kernel module `dm_crypt` via `cryptsetup` for the encryption.
-Further we use the Kubernetes secret mechanism for key storage, which can be further encrypted and guarded via appropriate permissions.
-An encrypted volume results in your data being encrypted while in transit as well as at rest, this also means that any backups taken from that volume are also encrypted.
+A volume in `Filesystem` or `Block` mode is supported to be encrypted by utilizing the Linux kernel module `dm_crypt` via `cryptsetup`. Further we use the Kubernetes secret mechanism for key storage, which can be further encrypted and guarded via appropriate permissions. An encrypted volume results in your data being encrypted while in transit as well as at rest, this also means that any backups taken from that volume are also encrypted.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/security/volume-encryption.md:6]: I reorganized and reworded the paragraph. Please check if the technical details are correct.

Longhorn supports volume encryption at the storage class level, which means that volumes in both `Filesystem` and `Block` mode can be encrypted while in transit and at rest. Moreover, backups taken from encrypted volumes are also encrypted. This is all accomplished through the Linux kernel module `dm_crypt`, the command-line utility `cryptsetup`, and Kubernetes Secrets. `dm_crypt` and `cryptsetup` handle the creation and management of encrypted devices, while Secrets (and related permissions) facilitate secure storage of encryption keys.

File: content/docs/1.6.0/advanced-resources/security/volume-encryption.md [hunk 2]
```diff
@@ -104,4 +102,5 @@ transparently used so no additional actions are needed from the user.
 Longhorn supports offline [expansion](../../../volumes-and-nodes/expansion) for encrypted volumes.
 
 # History
-Available since v1.2.0 [#1859](https://github.com/longhorn/longhorn/issues/1859)
+- Support Encryption for Volume in Filesystem Mode since v1.2.0 [#1859](https://github.com/longhorn/longhorn/issues/1859)
```
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/security/volume-encryption.md:105]: Please use the following wording:

Encryption of volumes in Filesystem mode available starting v1.2.0

File: content/docs/1.6.0/advanced-resources/security/volume-encryption.md [hunk 3]
```diff
@@ -104,4 +102,5 @@ transparently used so no additional actions are needed from the user.
...
+- Support Encryption for Volume in Block Mode since v1.6.0 [#4883](https://github.com/longhorn/longhorn/issues/4883)
```
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/security/volume-encryption.md:106]: Please use the following wording:

- Encryption of volumes in 'Block' mode available starting v1.6.0

**Review by jillian-maroket — COMMENTED** [review]
Please check the rewritten content.

File: content/docs/1.6.0/advanced-resources/security/volume-encryption.md (hunk 1 above)
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/security/volume-encryption.md:6]:
```suggestion
Longhorn supports volume encryption at the storage class level, which means that volumes in both Filesystem and Block mode can be encrypted while in transit and at rest. Moreover, backups taken from encrypted volumes are also encrypted. This is all accomplished through the Linux kernel module dm_crypt, the command-line utility cryptsetup, and Kubernetes Secrets. dm_crypt and cryptsetup handle the creation and management of encrypted devices, while Secrets (and related permissions) facilitate secure storage of encryption keys.
```

File: content/docs/1.6.0/advanced-resources/security/volume-encryption.md (hunk 2 above)
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/security/volume-encryption.md:105]:
```suggestion
- Encryption of volumes in `Filesystem` mode available starting v1.2.0
```

File: content/docs/1.6.0/advanced-resources/security/volume-encryption.md (hunk 3 above)
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/security/volume-encryption.md:106]:
```suggestion
- Encryption of volumes in `Block` mode available starting v1.6.0
```

**Review by jillian-maroket — COMMENTED** [review]
Let me know if the changes work.

**Comment by innobead** [comment]
@derekbit Please check the review comment from @jillian-maroket .

File: content/docs/1.6.0/advanced-resources/security/volume-encryption.md [hunk 4]
```diff
@@ -104,4 +102,6 @@ transparently used so no additional actions are needed from the user.
 Longhorn supports offline [expansion](../../../volumes-and-nodes/expansion) for encrypted volumes.
 
 # History
-Available since v1.2.0 [#1859](https://github.com/longhorn/longhorn/issues/1859)
+- Encryption of volumes in `Filesystem` mode available starting v1.2.0 ([#1859](https://github.com/longhorn/longhorn/issues/1859))
+
+- Support Encryption for Volume in Block Mode since v1.6.0 ([#4883](https://github.com/longhorn/longhorn/issues/4883))
```
[the remark this answers is not in the document]
Comment (derekbit) [reply content/docs/1.6.0/advanced-resources/security/volume-encryption.md]: Thank you.

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**